src/initialization.js

```diff
@@ -107,21 +107,27 @@ function initializeContainer(containerId, dataLayerName, previewUrl) {
 
     url += '&l=' + dataLayerName;
 
-    loadSnippet(url, dataLayerName);
+    loadSnippet(url, dataLayerName, settings);
 
     return true;
 }
 
-function loadSnippet(url, dataLayerName) {
+function loadSnippet(url, dataLayerName, settings) {
```

Comment:
This might come across as a nitpick, but I don't see any benefit passing in the entire `settings` object just for a single setting.  Unless we start needing several settings in the function, I would just add `preventAutoBlock` in as an argument.